Rakefile

```diff
@@ -29,14 +28,9 @@ namespace :assets do
   end
 end
 
-desc "Run RuboCop linting"
-task lint_ruby: :environment do
-  sh "bundle exec rubocop --format clang"
-end
-
-desc "Run Javascript and Sass linting"
-task lint_js_and_sass: :environment do
+desc "Linting for Ruby, JS and SASS"
+task lint: %i[rubocop environment] do
   sh "yarn run lint"
 end
```

Comment:
How does this task lint the Ruby? I can see it running the listing for the JavaScript and Sass via Yarn, but not how Rubocop is used.

Reply:
It's on line 32. As rubocop is a rake task we can define it as a dependant task that runs before the code in the lint block runs.

Not sure I explained this particularly well so let me know if it still doesn't make sense.

Reply:
Ah - the `%i[rubcocop environment]` is running the task - so if we wanted to add another dependent task it'd be `%[rubocop another_task environment]`?

Reply:
Yup, that's correct